feature-libs/product-configurator/rulebased/cpq/cpq-configurator-overview-normalizer.spec.ts

```diff
@@ -143,6 +146,19 @@ describe('CpqConfiguratorOverviewNormalizer', () => {
     expect(serviceUnderTest.convert(input).totalNumberOfIssues).toBe(3);
   });
 
+  it('should prepare price summary', () => {
+    expect(
+      serviceUnderTest.convert(input).priceSummary?.currentTotal?.formattedValue
+    ).toBe('$3,333.33');
+    expect(
+      serviceUnderTest.convert(input).priceSummary?.basePrice?.formattedValue
+    ).toBe('$1,000.00');
+    expect(
+      serviceUnderTest.convert(input).priceSummary?.selectedOptions
+        ?.formattedValue
+    ).toBe('$2,333.33');
+  });
```

Comment:
i guess its not intended to call convert 3 times here? if you call it once, the test gets better readable
```
const convertedPriceSummary =  serviceUnderTest.convert(input).priceSummary;
expected(convertedPriceSummary .currentToal.fomrattedValue).toBe.....
```

Reply:
Done.